specification/storagetasks/resource-manager/Microsoft.StorageTasks/stable/2023-01-01/storagetasks.json

```diff
@@ -0,0 +1,903 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "StorageTasksManagementClient",
+    "description": "The Azure Storage Tasks Management API.",
+    "version": "2023-01-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/providers/Microsoft.StorageTasks/operations": {
+      "get": {
+        "tags": [
+          "Storage Tasks Operations"
+        ],
+        "description": "Lists all of the available Storage Tasks Rest API operations.",
+        "operationId": "Operations_List",
+        "x-ms-examples": {
+          "OperationsList": {
+            "$ref": "./examples/misc/OperationsList.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
```

Comment:
>v1 [](http://example.com/codeflow?start=69&length=2)

Please use v5 types at all the places, instead of v1 types.

Reply:
I dont see any difference in v1 and v5 for these types. However I have updated the swagger with V5